cmd/chains.go

```diff
@@ -242,8 +213,35 @@ func cmdChainsShowDefault() *cobra.Command {
 		Short:   "show the configured default chain",
 		Args:    cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			fmt.Println(config.DefaultChain)
-			return nil
+			return config.GetDefaultClient().PrintObject(config.DefaultChain)
+		},
+	}
+	return cmd
+}
+
+func cmdChainsEditorDefault() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "editor",
+		Short: "Open Lens configuration in an editor",
+		Long: `Open Lens configuration in an editor. By default, command will spawn a vim window. You can 
+override the editor using the environment variable LENS_EDITOR. Please ensure $LENS_EDITOR points to 
+an editor in your path that can be called using $LENS_EDITOR <file-path>.`,
+		Args: cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			home, err := cmd.Flags().GetString("home")
+			if err != nil {
+				return err
+			}
+
+			editor := os.Getenv("LENS_EDITOR")
+			if editor == "" {
+				editor = "vim"
```

Comment:
nit: would be nice to fallback to the system default editor but 🤷 I like vim anyway

Reply:
I'd love to do this. How do you get the system default editor? I'm not finding anything online that lets you do it easily across Mac/Linux.

Reply:
According to [this](https://askubuntu.com/a/709034), the `EDITOR` env var should hold the system default. If it's blank, it uses `vi`. At least for Linux, according to https://askubuntu.com/a/709034

I switched it to check that and default to `vi`. Lmk if you have a better option though.